makefingerprint

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# makefingerprint generates a video perceptual hash for an input
+SCRIPTDIR=$(dirname $(which "${0}"))
+. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ;};
+SUFFIX="_signature"
+EXTENSION="xml"
+RELATIVEPATH="metadata"
+
+# Remove this section once 'signature' filter is integrated into official FFmpeg release.
+if [ ! -e ~/ffmpeg/ffmpeg ] ; then
```

Comment:
Up to you really!  It would be great to start collecting real world data sooner than later so I can start working on how to meaningfully crunch it.